eldritch/items/unique.py

```diff
@@ -56,3 +58,228 @@ def get_trigger(self, event, owner, state):
 
 def SwordOfGlory(idx):
   return Weapon("Sword of Glory", idx, "unique", {"magical": 6}, {}, 2, 8)
+
+
+def PallidMask(idx):
+  return Item("Pallid Mask", idx, "unique", {}, {"evade": 2}, None, 4)
+
+
+class AncientTablet(Item):
+  def __init__(self, idx):
+    super().__init__("Ancient Tablet", idx, "unique", {}, {}, None, 8)
+
+  def get_usable_interrupt(self, event, owner, state):
+    movement_cost = 3
+    if not isinstance(event, events.CityMovement) or event.character != owner or event.is_done():
+      return None
+
+    if self.exhausted:
+      return None
+
+    print(owner.movement_points)
+    if owner.movement_points < movement_cost:
+      return None
+
+    rolls = events.DiceRoll(owner, 2)
+    two_success = events.Draw(owner, "spells", 2, keep_count=2)
```

Comment:
What do you think of the wording on this? "For every success, draw 1 spell." Draw them both together? Or is this two separate draws? I think I agree with your current implementation (drawn together).

Reply:
I think it's just a single draw.  You don't want the Magician to be able to draw 2 extra cards.